webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -178,7 +179,10 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
         if (!summary || !this.selectedContact?.doc) {
           return;
         }
-
+        if (summary.errorStack){
+          this.summaryErrorStack = summary.errorStack;
```

Comment:
Based on my comment in the `contact-summary.service`, I think we could update this check to just be:

```suggestion
        if (summary instanceof Error){
          this.summaryErrorStack = summary.stack;
```

@latin-panda I would appreciate your wisdom on this situation. Maybe there is a better way to do this that I have missed.  Basically, we are subscribed to the store here to get the contact summary. Now we want to react (by setting the `summaryErrorStack`) if there is an error getting the contact summary.  The problem is that I cannot seem to figure out a clean way to do any actual error handling here. AFAICT, this code will only be triggered when calling `contactsActions.updateSelectedContactSummary`. Currently, I have suggested just passing the error object instead of the contact summary, but it feels like there has to be a better way....

Reply:
> The problem is that I cannot seem to figure out a clean way to do any actual error handling here. AFAICT, this code will only be triggered when calling contactsActions.updateSelectedContactSummary. Currently, I have suggested just passing the error object instead of the contact summary, but it feels like there has to be a better way.

@jkuester @jonathanbataire Yeah, it's kind of weird persisting errors in the Store. The error handling is cumbersome because we are [using effects](https://github.com/medic/cht-core/blob/3c3accebdfba8db7ad2c5261eeb75da79e7ad56c/webapp/src/ts/effects/contacts.effects.ts#L72) to load the contact information which is a multi-step process. 
IMO, it's better to do the contact load process in the `contact.service.ts`, and let the component (contact-content.controler.ts) receive and control the result; if it's successful, the component persists the contact info in the Store. If it fails, then display the error. However, this is a bigger refactor and perhaps out of scope for this feature. 
For the time being, adding a catch [here](https://github.com/medic/cht-core/blob/3c3accebdfba8db7ad2c5261eeb75da79e7ad56c/webapp/src/ts/effects/contacts.effects.ts#L163), as store using `this.contactsActions.updateSelectedContactSummary(summary)` seems enough as temporary solution. 
```
return this.contactSummaryService
      .get(selected.doc, selected.reports, selected.lineage, selected.targetDoc)
      .catch(error => {
        return { error };
      })
      .then(summary => {
        return this
          .verifySelectedContactNotChanged(contactId)
          .then(() => {
            this.contactsActions.setContactsLoadingSummary(false);
            return this.contactsActions.updateSelectedContactSummary(summary);
          });
      });
```